ladybug_comfort/utci.py

```diff
@@ -374,3 +165,250 @@ def fn(x):
         for key in missing_key:
             utci_inputs[key] = missing_val
     return utci_inputs
+
+
+def _utci_polynomial_function(ta, d_tr, vel, pa_pr):
```

Comment:
I think we can just call this `_utci_polynomial()`. It's already pretty obvious that it's a function given that it is written as such.